aptos/api/aux-ts/src/coin.ts

```diff
@@ -82,6 +82,14 @@ export enum FakeCoin {
  */
 export const ALL_FAKE_STABLES: FakeCoin[] = [FakeCoin.USDC, FakeCoin.USDT];
 
+export function fakeStableTypes(auxClient: AuxClient): string[] {
+  let types = [];
```

Comment:
this should prob be a linter config - but a tiny nit - types isn't being rebound to anything so it ought to be a `const`